packages/zipkin/test/httpServerInstrumentation.test.js

```diff
@@ -166,6 +118,89 @@ describe('Http Server Instrumentation', () => {
     });
   });
 
+  const samplingFlagCases = [
+        {headers: {}, hasAnnotations: null},
+        {headers: {'X-B3-Flags': '0'}, hasAnnotations: null},
```

Comment:
`hasAnnotations: null` means that we can not determine whether the annotation is recorded or not (depends on sampling policy). Although we could inject an `alwaysSampler` it is not the question of this test.